spec/system/user_uses_the_editor_spec.rb

```diff
@@ -57,7 +57,9 @@ def fill_markdown_with(content)
     end
   end
 
-  describe "Submitting an article", js: true do
+  describe "Submitting an article with v1 editor", js: true do
+    before { user.update(editor_version: "v1") }
```

Comment:
We want to be notified if this fails 🙂

```suggestion
    before { user.update!(editor_version: "v1") }
```